src/quo2/screens/new_messages_header.cljs

```diff
@@ -0,0 +1,54 @@
+(ns quo2.screens.new-messages-header
+  (:require [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [quo.previews.preview :as preview]
+            [quo.design-system.colors :as colors]
+            [quo2.components.new-messages-header :as new-messages-header]))
+
+(def descriptor [{:label   "Label"
+                  :key     :label
+                  :type    :text}
+                 {:label   "Color"
+                  :key     :color
+                  :type    :select
+                  :options [{:key   :primary
+                             :value "Primary"}
+                            {:key   :purple
+                             :value "Purple"}
+                            {:key   :indigo
+                             :value "Indigo"}
+                            {:key   :turquoise
+                             :value "Turquoise"}
+                            {:key   :blue
+                             :value "Blue"}
+                            {:key   :green
+                             :value "Green"}
+                            {:key   :yellow
+                             :value "yellow"}
+                            {:key   :orange
+                             :value "Orange"}
+                            {:key   :red
+                             :value "Red"}
+                            {:key   :pink
+                             :value "Pink"}
+                            {:key   :brown
+                             :value "Brown"}
+                            {:key   :beige
+                             :value "Beige"}]}])
+(defn cool-preview []
+  (let [state     (reagent/atom {:label "New messages"
```

Comment:
Maybe add "New messages" label to `en.json` translation file and fetch it with i18n

Reply:
Added